engine/settings/base.py

```diff
@@ -964,6 +964,8 @@ class BrokerTypes:
 
 ACKNOWLEDGE_REMINDER_TASK_EXPIRY_DAYS = os.environ.get("ACKNOWLEDGE_REMINDER_TASK_EXPIRY_DAYS", default=14)
 
+CUSTOM_RATELIMITS: str = json.loads(os.getenv("CUSTOM_RATELIMITS", "{}"))
```

Comment:
Do you mind add a comment & example of env. E.g should public or internal id be used.

Reply:
done